src/load_inputs/load_generators_data.jl

```diff
@@ -501,4 +496,47 @@ function load_vre_stor_data!(inputs_gen::Dict, setup::Dict, path::AbstractString
 		inputs_gen["dfVRE_STOR"] = DataFrame()
 	end
 	summarize_errors(error_strings)
-end
\ No newline at end of file
+end
+
+
+function process_piecewisefuelusage!(inputs::Dict, scale_factor)
+	gen_in = inputs["dfGen"]
+	inputs["PWFU_Max_Num_Segments"] = 0
+	inputs["THERM_COMMIT_PWFU"] = Int64[]
+
+	if any(occursin.(Ref("PWFU_"), names(gen_in)))
+		slope_mat = extract_matrix_from_dataframe(gen_in, "PWFU_Slope")
+		intercept_mat = extract_matrix_from_dataframe(gen_in, "PWFU_Intercept")
+		if size(slope_mat)[2] != size(intercept_mat)[2]
+			@error """ The number of slope and intercept columns used for piecewise fuel consumption must be equal
```

Comment:
This should probably call `error`, which actually stops the program, not just `@error` which prints a message. Same with below. You can always have both!

Reply:
Also, I find it's good to give the user some feedback: how many slope and intercept columns did the program find?